anaconda-mode.el

```diff
@@ -380,8 +380,11 @@ be bound."
   "Run `anaconda-mode' server.
 CALLBACK function will be called when `anaconda-mode-port' will
 be bound."
+  (if (not (file-directory-p anaconda-mode-installation-directory))
+      (make-directory anaconda-mode-installation-directory t))
   (setq anaconda-mode-process
         (pythonic-start-process :process anaconda-mode-process-name
+                                :cwd anaconda-mode-installation-directory ;; prevent conflicts with stdlib modules
```

Comment:
Can you add a guard around this to only set the cwd when `(pythonic-local-p)` is true? When editing over tramp the cwd needs to be on the remote host.

Reply:
yes, done.